.gitignore

```diff
@@ -1,47 +1,6 @@
-# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```

Comment:
The current .gitignore content must be kept. We should only add what's missing

Reply:
missing existing ignore entries re-added